cardano-ledger-core/src/Cardano/Ledger/BaseTypes.hs

```diff
@@ -103,54 +108,81 @@ type FixedPoint = Digits34
 fpPrecision :: FixedPoint
 fpPrecision = (10 :: FixedPoint) ^ (34 :: Integer)
 
+-- | Maximum precision possible for unit interval when backed by a 64bit
+type UnitScale = 19
+
+unitScale :: Int
```

Comment:
I was on the verge of calling it myself something along those lines. It just conflicts a little with Decimal terminology:

![image](https://user-images.githubusercontent.com/2333894/121664998-bd111400-cab0-11eb-9924-f24a6b1782d5.png)

For Decimal we call the Word64 (or rather RawUnit) precision, while scaling parameter `19` is the number of digits after the decimal point. (FYI I didn't come up with that terminology ;) )

I'll change it to `unitIntervalPrecision`. I don't think it will introduce any confusion.

Reply:
Something about `mantissa`?

Reply:
Done